src/SConscript

```diff
@@ -73,6 +73,20 @@ if not GetOption('disable_lib'):
     for s in symlinks:
         env.AddDistFile(env['ROC_SYSTEM_LIBDIR'], s)
 
+    if 'PKG_CONFIG_PATH' in env.Dictionary():
+        pc_target = env.GeneratePkgConfig(
+            build_dir='build',
+            filename='libroc.pc',
+            prefix=GetOption('prefix'),
+            libdir=env['ROC_SYSTEM_LIBDIR'],
+            name='roc',
+            desc='Real-time audio streaming over the network.',
+            url='https://roc-project.github.io',
+            version=env['ROC_VERSION'],
+            dependencies=env.get('PKG_CONFIG_DEPS', []),
+            libs=env.get('PKG_CONFIG_LIBS', []))
```

Comment:
I suggest to hide PKG_CONFIG_DEPS and PKG_CONFIG_LIBS inside GeneratePkgConfig (and remove dependencies and libs args), so that these variables will be an implementation detail.